docs/learningLab/introToLit/progSigning.md

```diff
@@ -1,16 +1,198 @@
 ---
-sidebar_position: 5
+sidebar_position: 4
 ---
-# 5. What can you build with Programmatic Signing
+# 4. Programmatic Signing
+![image](../../../static/img/ll_progSigning.png)
 
 As a distributed key management network, Lit provides developers with the ability to add programmable signing to their applications and wallets. These distributed wallets are known as Programmable Key Pairs (PKP) and the application logic that dictates when and why that key-pair will sign is known as a Lit Action.
 
+---
+
+## Objectives
```

Comment:
again thinking about the flow of information here. Perhaps considering:
1. Introducing PKPs
2. Introducing Lit Actions
3. How Lit Actions and PKPs work together
4. Use Cases

Moving into a 'working with PKPs and Lit Actions' section (or 'using PKPs and Lit Actions') and including the minting examples and the Lit Action examples